src/libs/SearchUtils.ts

```diff
@@ -469,8 +471,8 @@ function buildQueryStringFromFilters(filterValues: Partial<SearchAdvancedFilters
             Array.isArray(filterValue) &&
             filterValue.length > 0
         ) {
-            const filterValueArray = filterValues[filterKey] ?? [];
-            const keyInCorrectForm = (Object.keys(CONST.SEARCH.SYNTAX_FILTER_KEYS) as KeysOfFilterKeysObject[]).find((key) => CONST.SEARCH.SYNTAX_FILTER_KEYS[key] === filterKey);
+            const filterValueArray = Array.from(new Set<string>(filterValues[filterKey] ?? []));
```

Comment:
there is a tiny bit simpler way to create array from set
`const arr = [...(new Set())]` - I prefer this one because it uses destructuring and reads a bit better.